applications/launchpad/gui-react/src/components/Capitalize/index.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+const CapitalizeSpan = styled.span`
+  text-transform: capitalize;
+`
+
+const Capitalize = ({ children }: { children: string }) => (
```

Comment:
JSDoc and simple unit test could be useful for this component.

Reply:
this is my bad, I created it and then removed it as I no longer needed it in any of the other components :D I'll remove it in next PR #152